optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -79,16 +79,92 @@
       <artifactId>hibernate-entitymanager</artifactId>
       <scope>test</scope>
     </dependency>
+
+
     <dependency>
       <groupId>com.h2database</groupId>
       <artifactId>h2</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.btm</groupId>
+      <artifactId>btm</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
see my comment above. sure, if we create dependency on drools-jpa-persistence,  then it makes sense. but do we really want it?